### ISSUE TITLE
Form shows all validation messages by default / Form not working

### ISSUE THREAD

**Issue by grayspace** [2sic/app-mobius-forms#15]
Just installed the Mobius Forms app, and dragged a form onto the page. By default, and without inputting anything into the form, the validation alert boxes are shown (1st screenshot below). Browser refresh doesn't help, but I noticed they go away after clicking the manage content icon. They return after another browser refresh.

Not sure if related, or something else, but the form does not work for me. When the validation messages are shown as described above, the form does not submit at all. Nothing happens. When I click manage content, and the alert boxes go away, I can click Send, but I get an generic unknown error (2nd screenshot below).

DNN: v. 09.02.01 (533)
2sxc: v09.32.01-LTS
Mobius app: v02.00.01

![image](https://user-images.githubusercontent.com/5733194/45056280-060e2100-b058-11e8-9c72-6107f2719cc9.png)

![image](https://user-images.githubusercontent.com/5733194/45056921-d7914580-b059-11e8-95e5-8bf8bad418e2.png)

**Comment by iJungleboy** [comment]
So first of all I think it's something with CSS - my guess is that your DNN has cached the css and doesn't load the moebius CSS for some reason. That's why you're "already" seeing the errors. Could you check that? I seriously assume the errors have not been triggered at all - but are showing for some CSS reasons. Maybe you also have a special class overriding a setting. 

So please debug that. Validate
* [ ] that the CSS is loading (with F12 debugging)
* [ ] check in "inspect element" what the display-styles are, and where they are coming from

Your second error is probably that you edited the settings - thereby replacing defaults - and that these are not correct. You should see more details about this in the DNN event log. Please try that and give us feedback :)

**Comment by grayspace** [comment]
Debugged and saw that there were 404s on some 2sxc javascript files. I reinstalled 2sxc and Mobius forms and it now works as expected. One thing I noticed while reinstalling Mobius, it said it could not install because a folder named "Mobius Forms" already exists. So I just added a new folder called "MobiusForms" (no space), and after it installed I had no issues with the form again. Not sure if that was it, or I just had a bad install the first time. Anyway, it's working! Thanks.
